service/discovery/azure/storage.go

```diff
@@ -433,22 +433,22 @@ func (d *azureStorageDiscovery) discoverStorageAccounts() ([]voc.IsCloudResource
 			return res.Value
 		},
 		func(account *armstorage.Account) error {
-			activityLoggingAccount, activityLoggingBlob, activityLoggingFile, activityLoggingTable := d.getActivityLogging(account)
+			activityLoggingAccount, activityLoggingBlob, activityLoggingFile, activityLoggingTable, rawAccount, rawBlob, rawTable, rawFile := d.getActivityLogging(account)
```

Comment:
Ok for now, but in future/when migrating, I would maybe consider it doing it differently because for the sake of readability. First thought is to use a struct.
Also the rawXXX variables are a little bit confusing since they are not the raw values of the whole thing but only of the activity logging as far as I can tell. This would be also solved with a activity struct and putting everything inside.